docsets/openFrameworks/docset.json

```diff
@@ -10,6 +10,10 @@
         "of"
     ],
     "specific_versions": [
+        {
+            "version": "0.12.0",
+            "archive": "versions/0.12.0/openFrameworks.tgz"
```

Comment:
This file doesn't exist. You added `versions/0.12.0/openframeworks.tgz` instead.